src/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -279,12 +272,12 @@
         action.setCallback(this, callback);
         $A.enqueueAction(action);
 
-        // delete any contacts
-        var listConDelete = component.get("v.listConDelete");
-        listConDelete = this.addPrefixToListObjectFields(namespacePrefix, listConDelete);
-        action = component.get("c.deleteContacts");
+        // remove any contacts
+        var listConRemove = component.get("v.listConRemove");
+        listConRemove = this.addPrefixToListObjectFields(namespacePrefix, listConRemove);
```

Comment:
I was confused because you removed the namespace prefix processing from `HH_AddressMgrHelper` but not here.  So when sending sObjects to the server, namespaces have to be added, but because you're using plain objects within the component, the framework isn't doing anything tricky to add prefixes to the objects you query or pass in?

Since this is intentional and it works, I'm fine with it... but I don't totally understand it :)